app/assets/images/nhs-logo.svg

```diff
@@ -1,4 +1,4 @@
-<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 69.2 28">
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 69.2 28" width="60px" height="33px">
```

Comment:
If I remember right aren't these width and heights normally set as just integers ,`width="60" and height="33"`, as per the `image` element?

Reply:
It uses px as default, can change if you want? 

Ref: http://tutorials.jenkov.com/svg/svg-viewport-view-box.html#coordinate-system-units

Reply:
Oh cool - I didn't realise it accepted so much. As long as that is suitably idiomatic then I'm fine with that.

Reply:
For consistency, I've updated to remove the px :)

Reply:
Super 👍